download-maven-plugin/src/main/java/com/googlecode/download/maven/plugin/internal/WGet.java

```diff
@@ -176,7 +187,16 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             getLog().info("maven-download-plugin:wget skipped");
             return;
         }
-
+        
+        if (!StringUtils.isBlank(serverId) && (!StringUtils.isBlank(username) || !StringUtils.isBlank(password))) {
+            getLog().error("Specify either serverId or username/password, not both");
+            throw new MojoExecutionException("Specify either serverId or username/password, not both");
+        }
+        
+        if (settings == null) {
```

Comment:
I'll swap that out to a warn.  This is present because I had a spot of debugging trouble.